external/curl/curl_config.h

```diff
@@ -813,22 +813,22 @@
 #define SIZEOF_INT 4
 
 /* The size of `long', as computed by sizeof. */
-#define SIZEOF_LONG 8
+#define SIZEOF_LONG 4
 
 /* The size of `long long', as computed by sizeof. */
 /* #undef SIZEOF_LONG_LONG */
 
 /* The size of `off_t', as computed by sizeof. */
-#define SIZEOF_OFF_T 8
+#define SIZEOF_OFF_T 4
 
 /* The size of `short', as computed by sizeof. */
 #define SIZEOF_SHORT 2
 
 /* The size of `size_t', as computed by sizeof. */
-#define SIZEOF_SIZE_T 8
+#define SIZEOF_SIZE_T 4
```

Comment:
the size of 'size_t' depends on CONFIG_SMALL_MEMORY.
So it might be correct as follows:
#ifdef CONFIG_SMALL_MEMORY
#define SIZEOF_SIZE_T 2
#else
#define SIZEOF_SIZE_T 4
#endif

Reply:
@davidfather I applied your recommand. Thank you